### PR TITLE
Initial creation of folders and files

### LINKED ISSUE #1: initial folder setup

### PR DESCRIPTION
**Pull request by NessJade96**
The initial folder structure and contents are copied from a previous student mini project, make sure to check the code I have left behind matches our project as I made *some* changes :)

### CODE REVIEW

File: Main/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules
+.env
+.DS_Store
```
Comment (hayleyarodgers) [inline Main/.gitignore]: I think we should add `package-lock.json` to this list!

**Review by hayleyarodgers — COMMENTED** [review]
Incredible work setting up our repo with all of the files we're going to need 🙌🏼 I just left one comment on the gitignore file!

**Comment by dorisliu333** [comment]
removed the Main folder to avoid some structure problems when trying to deploy in the future and comment line the 'helper' function related code to make server work

File: Main/.gitignore (hunk 1 above)
Comment (NessJade96) [reply Main/.gitignore]: Good idea, I have added this to my latest PR. thank you :)

Comment (NessJade96) [reply Main/.gitignore]: I just did a quick google to check, and the package-lock.json is supposed to be in the git repo, so this ensures we are all using the same node modules/ dependencies versions. I will remove my latest PR including it in the gitignore. thanks! here is the link in you're interested: https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

**Review by dorisliu333 — APPROVED** [review]
nice!

**Pull request merged**